cloud/shared/bin/rotate_app_secret.py

```diff
@@ -0,0 +1,27 @@
+import importlib
+import os
+import sys
+from typing import List
+
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+
+
+def run(config: ConfigLoader, _params: List[str]):
+    rotate_file_py = os.path.join(
+        'cloud', config.get_cloud_provider(), 'bin', 'rotate_app_secret.py')
+    # TODO(#2741): remove the fork after we remove non-Python scripts
+    if os.path.exists(rotate_file_py):
+        rotate_module = importlib.import_module(
+            f'cloud.{config.get_cloud_provider()}.bin.rotate_app_secret')
+        if sys.stdin.isatty():
+            answer = input(
+                "WARNING: Rotating the app secret will invalidate all existing sessions. Any guest users with an unsubmitted application in progress will lose their application, and logged in users and admins will need to log in again. Are you sure you want to continue? [Y/n]: "
+            )
+            if answer.lower() in ['y', 'yes', '']:
```

Comment:
suggestion (non-blocking): Does including `''` mean that if they hit `enter` it will run? If so, I would remove that option so they don't accidentally trigger it by trying to exit the dialogue or something.

Reply:
That's my preference since I would prefer to accidentally exit the script than accidentally log everyone out.